fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.staging.codacy.org/gh/stagingcodacytestuser/CheckPrCommentsPublicProjectStaging/pullRequest?prid=19514)
```suggestion
(function ($, window) {
```